src/Inventory/Home.js

```diff
@@ -46,34 +36,48 @@ export default class Home extends Component {
 						color={colors.tint}
 						size={24}
 						underlayColor='transparent'
-						onPress={() => navigation.navigate('CreateList')}
+						onPress={() => navigation.navigate('CreateList', { loadList: params.loadList})}
```

Comment:
I'm not a fan of passing down props like this because it can get really messy; is there something preventing us from creating the function here?

Reply:
Nope, I was just trying to pass down the function so that it can navigate to the list right after creation, which didn't work. I'll remove it. And try again later.